### PR TITLE
ISS-44 login to account 

### PR DESCRIPTION
**Pull request by Hamilkar247**
example of adding logging to django https://learndjango.com/tutorials/django-login-and-logout-tutorial

Signed-off-by: Hamilkar247 <adonnibal96@gmail.com>

### CODE REVIEW

**Comment by Hamilkar247** [comment]
fixes #44

File: bostardecksweb/urls.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
     path('api/v1/', include('api_v1.urls')),
     path('core/', include('core.urls')),
     path('front/', include('front.urls')),
+    path('', include('front.urls')),
```
Comment (jlatusek) [inline bostardecksweb/urls.py:29]: by that you create two paths to every page in front app, I think that front app should be only available on root path, and you should remove line 27

File: front/templates/front/raw.html [hunk 2]
```diff
@@ -39,7 +39,7 @@ <h1>Bostar Deck</h1>
       <!-- unauthenticated raw.html-->
       <div class="contentttt class" style>
              <p>Nie jesteś zalogowany!</p>
-             <a href="login" class="btn btn-w-m btn-success">Zaloguj się !</a>
+             <a href="accounts/login" class="btn btn-w-m btn-success">Zaloguj się !</a>
```
Comment (jlatusek) [inline front/templates/front/raw.html:42]: Use django reverse url mechanism and don't hard code urls in templates

File: bostardecksweb/urls.py (hunk 1 above)
Comment (Hamilkar247) [reply bostardecksweb/urls.py:29]: ok ... :'(

File: front/templates/front/raw.html (hunk 2 above)
Comment (Hamilkar247) [reply front/templates/front/raw.html:42]: `django reverse url mechanism`

What do you mean?

I am not used this before - can you have sent me link on web where they use this mechanism?

File: bostardecksweb/urls.py (hunk 1 above)
Comment (Hamilkar247) [reply bostardecksweb/urls.py:28]: done

File: front/templates/front/raw.html (hunk 2 above)
Comment (Hamilkar247) [reply front/templates/front/raw.html]: done

**Comment by Hamilkar247** [comment]
@jlatusek

File: front/templates/front/raw.html (hunk 2 above)
Comment (jlatusek) [inline front/templates/front/raw.html]: hardcoded val

**Review by jlatusek — COMMENTED** [review]
LGTM

**Pull request merged**